blob/src/main/java/com/bazaarvoice/emodb/blob/BlobStoreModule.java

```diff
@@ -339,13 +339,13 @@ CacheRegistry provideCacheRegistry(CacheRegistry cacheRegistry) {
     @Provides @Singleton @TableChangesEnabled
     ValueStore<Boolean> provideTableChangesEnabled(@BlobStoreZooKeeper CuratorFramework curator,
                                                    LifeCycleRegistry lifeCycle) {
-        return lifeCycle.manage(new ZkValueStore<>(curator, "settings/table-changes-enabled", new ZkBooleanSerializer(), true));
+        return lifeCycle.manage(new ZkValueStore<>(curator, "/settings/table-changes-enabled", new ZkBooleanSerializer(), true));
```

Comment:
ZK path requires leading `/` in new versions.

Reply:
as zookeeper version was downgraded, should this to be reverted also?

Reply:
I believe it is actually curator, not ZK that enforces this. Additionally, I feel like it may be better to not rely on a leading `/` to be added for us anyway. If you really feel strongly about it than I can experiment with removing it.

Reply:
as it's only emo related thing and no other team rely on it we can leave it with leading `/`

Reply:
Also, to clarify, the leading / does not change the location of the value. The library used to just add it for us